src/web/file.rs

```diff
@@ -73,6 +74,7 @@ impl IntoResponse for File {
                     &self.0.date_updated.format("%a, %d %b %Y %T %Z").to_string(),
                 ),
             ],
+            Extension(CachePolicy::ForeverInCdnAndBrowser),
```

Comment:
this was a leftover where the new `IntoResponse` didn't set the cache policy, as  `File.serve` does.

Reply:
I also added a test for this